src/styles/variables.scss

```diff
@@ -22,6 +22,10 @@
 	--dswp-surface-color-primary-button-default: var(--bcds-surface-color-primary-button-default);
 	--dswp-banner-background-dark: #292929;
 	--dswp-theme-primary-gold: var(--bcds-theme-primary-gold);
+	--dswp-icons-color-info: var(--bcds-icons-color-info);
```

Comment:
Do we want these in the theme.json in the color palette too?

Reply:
for now, no.